integ-test/src/test/java/org/opensearch/sql/ppl/FillNullCommandIT.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.ppl;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_CALCS;
+import static org.opensearch.sql.util.MatcherUtils.rows;
+import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
+
+import java.io.IOException;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Test;
+
+public class FillNullCommandIT extends PPLIntegTestCase {
+  @Override
+  public void init() throws IOException {
+    loadIndex(Index.CALCS);
+  }
+
+  @Test
+  public void testFillNullSameValueOneField() throws IOException {
+    JSONObject result =
+        executeQuery(
+            String.format(
+                "source=%s | fields str2, num0 | fillnull with -1 in num0", TEST_INDEX_CALCS));
```

Comment:
I'd show unchanged and filled column in the test (and in doctests and in PR description as well)

Reply:
There is an integration test that updates one field, but not the other (`testFillNullWithOtherField`).

There is a test in the docs as well (the first query in `fillnull.rst`).

The description for this PR has been updated with a sample query.